specification/hybridcloud/resource-manager/Microsoft.HybridCloud/preview/2022-11-01-preview/hybridcloud.json

```diff
@@ -0,0 +1,992 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-11-01-preview",
+    "title": "hybridcloud",
```

Comment:
>hybridcloud [](http://example.com/codeflow?start=14&length=11)

Should capitalize as Hybrid Cloud, HybridCloud, or Hybrid cloud

Reply:
Fixed